service/src/test/java/uk/nhs/adaptors/gp2gp/common/task/TaskErrorHandlerTest.java

```diff
@@ -50,13 +47,9 @@ class TaskErrorHandlerTest {
     @InjectMocks
     private TaskErrorHandler taskErrorHandler;
 
```

Comment:
if "when(taskDefinition.getTaskType()).thenReturn(GET_GPC_STRUCTURED);" happens to be needed in every test I'd keep it the common setup (@BeforeEach) method